ChilledCafe/Views/DetailPages/BODetailView.swift

```diff
@@ -86,7 +86,12 @@ struct BODetailView: View {
                     HStack {
                         Spacer()
                         if cafe.ar {
-                            arButtonView
+                            NavigationLink(destination:
+                                            ARMainView()
+                            )
+                            {
+                                arButtonView
+                            }
```

Comment:
```suggestion
                            NavigationLink(destination: ARMainView()) {
                                arButtonView
                            }
```
제안사항 ) 코드 모양을 이렇게 바꿔보면 어떨까요?